src/nanotron/config/parallelism_config.py

```diff
@@ -20,6 +20,7 @@ class ParallelismArgs:
         dp: Number of DP replicas
         pp: Number of PP stages
         tp: Number of TP replicas
+        expert_parallel_size: Number of expert parallel replicas (used only for MoEs)
```

Comment:
Isn't expert_parallel_size should be the number of experts per tp rank?

Reply:
Not quite, expert parallelism is orthogonal to TP. for example you can have 1 expert sharded along 2 tp ranks